README.md

```diff
@@ -470,4 +471,30 @@ Success! The configuration is valid.
 
 6. Finally, to get the feature into the production environment, the same pull request, review, and merge process will occur. The only difference in this situation is merging the **qa** branch into the **prod** branch.
 
-15. After the merge to prod finishes and is the issue is considered complete, the GitHub isssue can be closed and the development branch can be deleted. When the development branch is deleted, a GitHub Action will be triggered to delete the corresponding PingOne Environment leaving only the **qa** and **prod** environments relevant to this example.
+7. After the merge to prod finishes and is the issue is considered complete, the GitHub issue can be closed and the development branch can be deleted. When the development branch is deleted, a GitHub Action will be triggered to delete the corresponding PingOne Environment leaving only the **qa** and **prod** environments relevant to this example.
+
+## Cleanup Instructions
+
+After practicing with this example repository, you may wish to completely remove the environments it creates and manages from your PingOne account. In testing this guide, a potential problem with permissions was identified.  The following steps will guide you through the process of removing the environments safely until an automated solution is available.
```

Comment:
how about: 
After practicing with this example repository, you may wish to completely remove the environments it creates and manages from your PingOne account. To avoid scenarios of leftover, unmanaged permissions, the following steps will guide you through the process of removing the environments _safely_ until an automated solution is available.